src/containers/LoginScreen.js

```diff
@@ -7,21 +7,44 @@ class LoginScreen extends Component {
     email: '',
     password: ''
   }
+
   handleChange = e => {
     let change = {}
     change[e.target.name] = e.target.value
     this.setState(change)
   }
 
   createAccount = e => {
-    console.log(this.state)
+    this.fetchAccount('signup')
```

Comment:
Rather then just passing the string, let's pass the entire URL

Reply:
login and create methods now pass the entire url. also removed the console.log